src/Homework5/task3.py

```diff
@@ -0,0 +1,22 @@
+"""Реализовать функцию get_ranges которая получает на
+   вход непустой список неповторяющихся целых чисел,
+   отсортированных по возрастанию, которая этот список
+   “сворачивает”
+   get_ranges([0, 1, 2, 3, 4, 7, 8, 10]) // "0-4,7-8,10"
+   get_ranges([4,7,10]) // "4,7,10"
+   get_ranges([2, 3, 8, 9]) // "2-3,8-9"
+"""
+
+
+def get_ranges(a):
```

Comment:
`0-4,7-8,10-10` - your solution  result for `0, 1, 2, 3, 4, 7, 8, 10`. 
Expected -  `0-4,7-8,10`